src/fu-release.c

```diff
@@ -323,6 +323,10 @@ fu_release_load_test_result(FuRelease *self, XbNode *n, GError **error)
 	if (custom != NULL) {
 		for (guint i = 0; i < custom->len; i++) {
 			XbNode *c = g_ptr_array_index(custom, i);
+			if (g_strcmp0(xb_node_get_attr(c, "key"), "FromOEM") == 0) {
```

Comment:
Isn't this named improperly? It's more like from a trusted source. For example if Dell shipped a dock update but it was validated by say Google in ChromeOS and Google happened to push a signed report it should be given some weight just like this I would think.

Reply:
The logic I was aiming for was to have something like `DistroId:chromeos,VendorId:$SELF;DistroId:chromeos,VendorId:19` i.e. a trusted report is anything by the original OEM, or by the Google QA team.